deb-get

```diff
@@ -656,7 +656,7 @@ function update_repos() {
     local WGET_VERBOSITY="-q"
 
     if [[ " $* " == *' --quiet '* ]]  ; then
-         CURL_VERBOSITY="-Ss"
+         CURL_VERBOSITY="-Ss -q"
```

Comment:
The `-q` option doesn't quiet curl, `-s` does.

The `-q` option tells curl to not read the user's `.curlrc` file. So, imho, that option should be added to all the curl calls (or even better, passed via an environment variable because some users may want curl to read it's config, so `export CURL_OPTS='-q'`)

Reply:
renamed curl options variable and moved to long naming
also moved --disable to first option because it has to be first